cross/fish/Makefile

```diff
@@ -0,0 +1,20 @@
+PKG_NAME = fish
+PKG_VERS = 3.1.2
+PKG_EXT = tar.gz
+PKG_DIST_NAME = $(PKG_NAME)-$(PKG_VERS).$(PKG_EXT)
+PKG_DIST_SITE = https://github.com/fish-shell/fish-shell/releases/download/$(PKG_VERS)
+PKG_DIR = $(PKG_NAME)-$(PKG_VERS)
+
+DEPENDS = cross/ncurses cross/pcre2
+OPTIONAL_DEPENDS = cross/bzip2 cross/readline cross/zlib
+
+HOMEPAGE = https://fishshell.com
+COMMENT = fish is a smart and user-friendly command line shell for Linux, macOS, and the rest of the family.
+LICENSE = Custom
+
+UNSUPPORTED_ARCHS = $(ARMv5_ARCHS) $(PPC_ARCHS)
+
+# fish needs pcre2-32
+ENV += PCRE2_CLI_FULL=1
+
```

Comment:
with updates of 628c0bbf53b4af650cd599d4d0c24bc456f0bb21 this is not required anymore, as pcre2-32 is always included in cross/pcre2
ENV += PCRE2_CLI_FULL=1 did not work (I had to export PCRE2_CLI_FULL=1)